app/controllers/album.js

```diff
@@ -56,7 +63,18 @@ exports.buyAlbums = (req, res, next) => {
 
 exports.getAlbumsList = (req, res, next) => {
   logger.info(`getAlbumsList method start, request methods: ${req.method}, endpoint: ${req.path}`);
-  return findAllAlbumsBy(req.params)
+  return findAllAlbumsBy({ condition: req.params, attributes: ['id', 'title'] })
     .then(albums => res.send({ albums }))
     .catch(next);
 };
+
+exports.getAlbumPhotosList = (req, res, next) => {
+  const { id } = req.params;
+  logger.info(
+    `getAlbumPhotosList method start, request methods: ${req.method}, endpoint: ${req.path}, id: ${id}`
+  );
+  const user = decodedToken(req.headers.token);
+  return getAlbumPhotos({ id, userId: user.id })
+    .then(albumPhotos => res.send({ albumPhotos }))
```

Comment:
do you know the difference between doing
`res.send(json)` and `res.send({json})`?

Reply:
`res.send(json)` The answer would be delivering as it comes from the service, in this case an array.
`res.send({json})` The answer would be delivering an object with the **albumPhotos** property and this in turn contains the array that delivers the service.